tests/operators/models/test_transient_steady_state_with_preparation.py

```diff
@@ -66,9 +66,9 @@ def test_transient_steady_state_shape(parameter_shape, contrast_dim_shape, signa
     """Test correct signal shapes."""
     (sampling_time,) = create_parameter_tensor_tuples(contrast_dim_shape, number_of_tensors=1)
     if len(parameter_shape) == 1:
-        repetition_time = 5
-        m0_scaling_preparation = 1
-        delay_after_preparation = 0.01
+        repetition_time: float | torch.Tensor = 5
```

Comment:
after adding the return type, this is needed for mypy to recognize that it can be either float (this branch) or tensor (the other branch). without it, it will complain on the other branch, as it will recognise the variables as only floats.